repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -47,29 +50,45 @@ def _prep_repository_access(context, target_userspace):
     context.copytree_from('/etc/yum.repos.d', os.path.join(target_userspace, 'etc', 'yum.repos.d'))
 
 
-def get_os_variant():
-    """
-    Retrieves the OS variant from the first matching OSReleaseFacts message. Returns an empty string otherwise.
-    """
-    for msg in api.consume(OSReleaseFacts):
-        if msg.variant_id:
-            return msg.variant_id
-    return ''
-
-
 def _get_product_certificate_path():
     """
     Retrieves the required / used product certificate for RHSM.
     """
-    sys_var = get_os_variant()
-    var_prodcert = {'server': '479.pem'}
-    if sys_var not in var_prodcert:
-        raise StopActorExecutionError(
-            message=('Failed to to retrieve Product Cert file.'
-                     'Product cert file not available for System Variant \'{}\'.'.format(sys_var))
-        )
+    config = next(api.consume(IPUConfig), None)
+    variant = config.os_release.variant_id
+    architecture = config.architecture
+    tgt_version = config.version.target
+    # TODO: so far only base channel is available in rhel8
+    tgt_channel = 'base'
+    certs_folder = api.get_common_folder_path(PROD_CERTS_FOLDER)
+
+    prod_certs = {
+        'server': {
+            'x86_64': {
+                'base': '479.pem',
```

Comment:
Instead of `base` I think we should rather use `standard` - that's the proper wording here.

Reply:
As per https://access.redhat.com/articles/rhel-eus#c4, `base` is the right term to use here.